### PR TITLE
Tweaked resource table to be easier debuggable

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

File: common/src/main/java/org/jboss/hal/testsuite/fragment/shared/table/TooFewColumnsException.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.jboss.hal.testsuite.fragment.shared.table;
+
+public class TooFewColumnsException extends RuntimeException {
+
+    public TooFewColumnsException(String message) {
```
Comment (zebrik) [inline common/src/main/java/org/jboss/hal/testsuite/fragment/shared/table/TooFewColumnsException.java]: Do you plan to use this constructor? Please remove it if not.

**Review by zebrik — APPROVED** [review]
Just one optional suggestion.

File: common/src/main/java/org/jboss/hal/testsuite/fragment/shared/table/TooFewColumnsException.java (hunk 1 above)
Comment (honza-kasik) [reply common/src/main/java/org/jboss/hal/testsuite/fragment/shared/table/TooFewColumnsException.java]: Fixed in amended commit, thanks!

**Comment by zebrik** [comment]
LGTM. Thx.

**Pull request merged**